subprojects/docs/src/docs/release/notes.md

```diff
@@ -10,6 +10,8 @@ Include only their name, impactful features should be called out separately belo
 -->
 We would like to thank the following community members for their contributions to this release of Gradle:
 
+[Herbert von Broeuschmeul](https://github.com/HvB)
```

Comment:
❌ In the upgrading_version_X.adoc file please list this change under `Potential breaking changes` to let other users know of the altered behavior.